!SESSION 2019-05-15 20:58:03.259 -----------------------------------------------
eclipse.buildId=4.11.0.I20190307-0500
java.version=11.0.2
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=de_DE
Framework arguments:  -product org.eclipse.epp.package.java.product
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.java.product

!ENTRY org.eclipse.egit.core 1 0 2019-05-15 20:58:30.215
!MESSAGE Using Apache MINA sshd as ssh client.

!ENTRY org.eclipse.ui 4 4 2019-05-15 20:58:34.085
!MESSAGE Referenced part does not exist yet: org.eclipse.ui.views.TaskList.

!ENTRY org.eclipse.egit.ui 2 0 2019-05-15 20:58:44.838
!MESSAGE Warning: The environment variable HOME is not set. The following directory will be used to store the Git
user global configuration and to define the default location to store repositories: 'C:\Users\Jannik'. If this is
not correct please set the HOME environment variable and restart Eclipse. Otherwise Git for Windows and
EGit might behave differently since they see different configuration options.
This warning can be switched off on the Team > Git > Confirmations and Warnings preference page.

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-15 20:58:55.618
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-15 20:58:55.623
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-15 20:58:55.631
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-15 20:58:55.748
!MESSAGE an error occurred while performing a STUN discovery: ICMP Port Unreachable (StunServiceImpl.java:155)
!STACK 0
java.net.PortUnreachableException: ICMP Port Unreachable
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketReceiveOrPeekData(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.receive0(DualStackPlainDatagramSocketImpl.java:124)
	at java.base/java.net.AbstractPlainDatagramSocketImpl.receive(AbstractPlainDatagramSocketImpl.java:181)
	at java.base/java.net.DatagramSocket.receive(DatagramSocket.java:814)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery.performStunDiscovery(StunServiceImpl.java:284)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery.run(StunServiceImpl.java:151)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-15 20:59:05.614
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.77.1:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-15 20:59:05.616
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /169.254.181.9:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-15 20:59:05.617
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.38.1:0 (StunServiceImpl.java:420)
!SESSION 2019-05-16 16:28:37.182 -----------------------------------------------
eclipse.buildId=4.11.0.I20190307-0500
java.version=11.0.2
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=de_DE
Framework arguments:  -product org.eclipse.epp.package.java.product
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.java.product

!ENTRY org.eclipse.egit.core 1 0 2019-05-16 16:28:47.399
!MESSAGE Using Apache MINA sshd as ssh client.

!ENTRY org.eclipse.egit.ui 2 0 2019-05-16 16:28:58.634
!MESSAGE Warning: The environment variable HOME is not set. The following directory will be used to store the Git
user global configuration and to define the default location to store repositories: 'C:\Users\Jannik'. If this is
not correct please set the HOME environment variable and restart Eclipse. Otherwise Git for Windows and
EGit might behave differently since they see different configuration options.
This warning can be switched off on the Team > Git > Confirmations and Warnings preference page.

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-16 16:29:02.384
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-16 16:29:02.562
!MESSAGE an error occurred while performing a STUN discovery: ICMP Port Unreachable (StunServiceImpl.java:155)
!STACK 0
java.net.PortUnreachableException: ICMP Port Unreachable
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketReceiveOrPeekData(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.receive0(DualStackPlainDatagramSocketImpl.java:124)
	at java.base/java.net.AbstractPlainDatagramSocketImpl.receive(AbstractPlainDatagramSocketImpl.java:181)
	at java.base/java.net.DatagramSocket.receive(DatagramSocket.java:814)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery.performStunDiscovery(StunServiceImpl.java:284)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery.run(StunServiceImpl.java:151)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-16 16:29:02.635
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-16 16:29:02.638
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-16 16:29:12.382
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.77.1:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-16 16:29:12.403
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.38.1:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-16 16:29:12.557
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /169.254.181.9:0 (StunServiceImpl.java:420)

!ENTRY org.eclipse.jdt.ui 4 10001 2019-05-16 19:11:17.011
!MESSAGE Internal Error
!STACK 0
org.eclipse.jface.text.BadLocationException
	at org.eclipse.jface.text.AbstractDocument.getChar(AbstractDocument.java:782)
	at org.eclipse.core.internal.filebuffers.SynchronizableDocument.getChar(SynchronizableDocument.java:141)
	at org.eclipse.jdt.internal.ui.text.java.hover.JavaSourceHover.getBracketHoverInfo(JavaSourceHover.java:179)
	at org.eclipse.jdt.internal.ui.text.java.hover.JavaSourceHover.getHoverInfo(JavaSourceHover.java:140)
	at org.eclipse.jdt.internal.ui.text.java.hover.JavaSourceHover.getHoverInfo2(JavaSourceHover.java:120)
	at org.eclipse.jdt.internal.ui.text.java.hover.BestMatchHover.getHoverInfo2(BestMatchHover.java:167)
	at org.eclipse.jdt.internal.ui.text.java.hover.BestMatchHover.getHoverInfo2(BestMatchHover.java:133)
	at org.eclipse.jdt.internal.ui.text.java.hover.JavaEditorTextHoverProxy.getHoverInfo2(JavaEditorTextHoverProxy.java:89)
	at org.eclipse.jface.text.TextViewerHoverManager$4.run(TextViewerHoverManager.java:169)

!ENTRY org.eclipse.debug.core 4 120 2019-05-16 22:07:40.629
!MESSAGE Variable references empty selection: ${project_loc}

!ENTRY org.eclipse.debug.core 4 120 2019-05-16 22:07:43.619
!MESSAGE Variable references empty selection: ${project_loc}

!ENTRY org.eclipse.debug.core 4 120 2019-05-16 22:07:48.407
!MESSAGE Variable references empty selection: ${project_loc}

!ENTRY org.eclipse.ui 4 4 2019-05-16 22:41:26.626
!MESSAGE Referenced part does not exist yet: org.eclipse.ui.views.TaskList.

!ENTRY org.eclipse.core.resources 4 566 2019-05-16 22:51:16.934
!MESSAGE Workspace was not properly initialized or has already shutdown.

!ENTRY org.eclipse.core.jobs 4 1 2019-05-16 22:51:16.943
!MESSAGE Worker thread ended job: Refreshing workspace(10), but still holds rule: ThreadJob(Refreshing workspace(10),[F/MDKExample/eclipse/.metadata/.plugins/org.eclipse.core.resources/.history])
!SESSION 2019-05-17 16:05:51.480 -----------------------------------------------
eclipse.buildId=4.11.0.I20190307-0500
java.version=11.0.2
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=de_DE
Framework arguments:  -product org.eclipse.epp.package.java.product
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.java.product

!ENTRY org.eclipse.egit.core 1 0 2019-05-17 16:06:11.540
!MESSAGE Using Apache MINA sshd as ssh client.

!ENTRY org.eclipse.egit.ui 2 0 2019-05-17 16:06:28.730
!MESSAGE Warning: The environment variable HOME is not set. The following directory will be used to store the Git
user global configuration and to define the default location to store repositories: 'C:\Users\Jannik'. If this is
not correct please set the HOME environment variable and restart Eclipse. Otherwise Git for Windows and
EGit might behave differently since they see different configuration options.
This warning can be switched off on the Team > Git > Confirmations and Warnings preference page.

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-17 16:06:33.442
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-17 16:06:33.450
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-17 16:06:33.455
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-17 16:06:43.440
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.178.20:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-17 16:06:43.458
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.77.1:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-17 16:06:43.463
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /169.254.181.9:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-17 16:06:43.467
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.38.1:0 (StunServiceImpl.java:420)

!ENTRY org.eclipse.ui 4 0 2019-05-17 20:44:08.899
!MESSAGE Unhandled event loop exception
!STACK 0
org.eclipse.swt.SWTException: Widget is disposed
	at org.eclipse.swt.SWT.error(SWT.java:4699)
	at org.eclipse.swt.SWT.error(SWT.java:4614)
	at org.eclipse.swt.SWT.error(SWT.java:4585)
	at org.eclipse.swt.widgets.Widget.error(Widget.java:451)
	at org.eclipse.swt.widgets.Widget.checkWidget(Widget.java:350)
	at org.eclipse.swt.widgets.Composite.getChildren(Composite.java:469)
	at org.eclipse.epp.internal.mpc.ui.wizards.MarketplaceDropAdapter.lambda$0(MarketplaceDropAdapter.java:136)
	at org.eclipse.swt.widgets.RunnableLock.run(RunnableLock.java:40)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:185)
	at org.eclipse.swt.widgets.Display.runAsyncMessages(Display.java:3919)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3550)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1173)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1062)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:155)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:644)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:566)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:150)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:155)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:203)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:137)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:107)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:400)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:255)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:661)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:597)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1476)

!ENTRY org.eclipse.ui 4 0 2019-05-17 20:44:08.958
!MESSAGE Unhandled event loop exception
!STACK 0
org.eclipse.swt.SWTException: Widget is disposed
	at org.eclipse.swt.SWT.error(SWT.java:4699)
	at org.eclipse.swt.SWT.error(SWT.java:4614)
	at org.eclipse.swt.SWT.error(SWT.java:4585)
	at org.eclipse.swt.widgets.Widget.error(Widget.java:451)
	at org.eclipse.swt.widgets.Widget.checkWidget(Widget.java:350)
	at org.eclipse.swt.widgets.Composite.getChildren(Composite.java:469)
	at org.eclipse.epp.internal.mpc.ui.wizards.MarketplaceDropAdapter.lambda$0(MarketplaceDropAdapter.java:136)
	at org.eclipse.swt.widgets.RunnableLock.run(RunnableLock.java:40)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:185)
	at org.eclipse.swt.widgets.Display.runAsyncMessages(Display.java:3919)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3550)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1173)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1062)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:155)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:644)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:566)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:150)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:155)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:203)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:137)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:107)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:400)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:255)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:661)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:597)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1476)

!ENTRY org.eclipse.jdt.ui 4 10001 2019-05-17 21:02:25.944
!MESSAGE Internal Error
!STACK 1
Java Model Exception: Java Model Status [loadComplete() [in TIGMod [in [Working copy] TIGMod.java [in de.maltesermailo.tigmod [in src/main/java [in MDKExample]]]]] does not exist]
	at org.eclipse.jdt.internal.core.JavaElement.newNotPresentException(JavaElement.java:573)
	at org.eclipse.jdt.internal.core.JavaElement.openWhenClosed(JavaElement.java:607)
	at org.eclipse.jdt.internal.core.JavaElement.getElementInfo(JavaElement.java:326)
	at org.eclipse.jdt.internal.core.JavaElement.getElementInfo(JavaElement.java:312)
	at org.eclipse.jdt.internal.core.SourceRefElement.getSourceRange(SourceRefElement.java:233)
	at org.eclipse.jdt.internal.core.Member.getJavadocRange(Member.java:301)
	at org.eclipse.jdt.internal.ui.text.javadoc.JavadocContentAccess2.getHTMLContentFromSource(JavadocContentAccess2.java:664)
	at org.eclipse.jdt.internal.ui.text.javadoc.JavadocContentAccess2.getHTMLContent(JavadocContentAccess2.java:540)
	at org.eclipse.jdt.internal.ui.infoviews.JavadocView.getJavadocHtml(JavadocView.java:1109)
	at org.eclipse.jdt.internal.ui.infoviews.JavadocView.computeInput(JavadocView.java:894)
	at org.eclipse.jdt.internal.ui.infoviews.AbstractInfoView$2.run(AbstractInfoView.java:761)
!SUBENTRY 1 org.eclipse.jdt.core 4 969 2019-05-17 21:02:25.944
!MESSAGE loadComplete() [in TIGMod [in [Working copy] TIGMod.java [in de.maltesermailo.tigmod [in src/main/java [in MDKExample]]]]] does not exist

!ENTRY org.eclipse.ui 4 0 2019-05-17 21:05:53.367
!MESSAGE Unhandled event loop exception
!STACK 0
org.eclipse.swt.SWTException: Widget is disposed
	at org.eclipse.swt.SWT.error(SWT.java:4699)
	at org.eclipse.swt.SWT.error(SWT.java:4614)
	at org.eclipse.swt.SWT.error(SWT.java:4585)
	at org.eclipse.swt.widgets.Widget.error(Widget.java:451)
	at org.eclipse.swt.widgets.Widget.checkWidget(Widget.java:350)
	at org.eclipse.swt.widgets.Composite.getChildren(Composite.java:469)
	at org.eclipse.epp.internal.mpc.ui.wizards.MarketplaceDropAdapter.lambda$0(MarketplaceDropAdapter.java:136)
	at org.eclipse.swt.widgets.RunnableLock.run(RunnableLock.java:40)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:185)
	at org.eclipse.swt.widgets.Display.runAsyncMessages(Display.java:3919)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3550)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1173)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1062)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:155)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:644)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:566)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:150)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:155)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:203)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:137)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:107)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:400)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:255)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:661)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:597)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1476)

!ENTRY org.eclipse.ui 4 0 2019-05-17 21:38:29.591
!MESSAGE Unhandled event loop exception
!STACK 0
org.eclipse.swt.SWTException: Widget is disposed
	at org.eclipse.swt.SWT.error(SWT.java:4699)
	at org.eclipse.swt.SWT.error(SWT.java:4614)
	at org.eclipse.swt.SWT.error(SWT.java:4585)
	at org.eclipse.swt.widgets.Widget.error(Widget.java:451)
	at org.eclipse.swt.widgets.Widget.checkWidget(Widget.java:350)
	at org.eclipse.swt.widgets.Composite.getChildren(Composite.java:469)
	at org.eclipse.epp.internal.mpc.ui.wizards.MarketplaceDropAdapter.lambda$0(MarketplaceDropAdapter.java:136)
	at org.eclipse.swt.widgets.RunnableLock.run(RunnableLock.java:40)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:185)
	at org.eclipse.swt.widgets.Display.runAsyncMessages(Display.java:3919)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3550)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1173)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1062)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:155)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:644)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:566)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:150)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:155)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:203)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:137)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:107)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:400)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:255)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:661)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:597)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1476)

!ENTRY org.eclipse.ui 4 0 2019-05-17 21:38:29.594
!MESSAGE Unhandled event loop exception
!STACK 0
org.eclipse.swt.SWTException: Widget is disposed
	at org.eclipse.swt.SWT.error(SWT.java:4699)
	at org.eclipse.swt.SWT.error(SWT.java:4614)
	at org.eclipse.swt.SWT.error(SWT.java:4585)
	at org.eclipse.swt.widgets.Widget.error(Widget.java:451)
	at org.eclipse.swt.widgets.Widget.checkWidget(Widget.java:350)
	at org.eclipse.swt.widgets.Composite.getChildren(Composite.java:469)
	at org.eclipse.epp.internal.mpc.ui.wizards.MarketplaceDropAdapter.lambda$0(MarketplaceDropAdapter.java:136)
	at org.eclipse.swt.widgets.RunnableLock.run(RunnableLock.java:40)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:185)
	at org.eclipse.swt.widgets.Display.runAsyncMessages(Display.java:3919)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3550)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1173)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1062)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:155)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:644)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:566)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:150)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:155)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:203)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:137)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:107)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:400)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:255)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:661)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:597)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1476)

!ENTRY org.eclipse.jdt.ui 4 10001 2019-05-17 21:40:16.771
!MESSAGE Internal Error
!STACK 1
Java Model Exception: Java Model Status [on [in BasicStone [in [Working copy] BasicStone.java [in de.maltesermailo.tigmod.stones [in src/main/java [in MDKExample]]]]] does not exist]
	at org.eclipse.jdt.internal.core.JavaElement.newNotPresentException(JavaElement.java:573)
	at org.eclipse.jdt.internal.core.JavaElement.openWhenClosed(JavaElement.java:607)
	at org.eclipse.jdt.internal.core.JavaElement.getElementInfo(JavaElement.java:326)
	at org.eclipse.jdt.internal.core.JavaElement.getElementInfo(JavaElement.java:312)
	at org.eclipse.jdt.internal.core.SourceRefElement.getSourceRange(SourceRefElement.java:233)
	at org.eclipse.jdt.internal.core.Member.getJavadocRange(Member.java:301)
	at org.eclipse.jdt.internal.ui.text.javadoc.JavadocContentAccess2.getHTMLContentFromSource(JavadocContentAccess2.java:664)
	at org.eclipse.jdt.internal.ui.text.javadoc.JavadocContentAccess2.getHTMLContent(JavadocContentAccess2.java:540)
	at org.eclipse.jdt.internal.ui.infoviews.JavadocView.getJavadocHtml(JavadocView.java:1109)
	at org.eclipse.jdt.internal.ui.infoviews.JavadocView.computeInput(JavadocView.java:894)
	at org.eclipse.jdt.internal.ui.infoviews.AbstractInfoView$2.run(AbstractInfoView.java:761)
!SUBENTRY 1 org.eclipse.jdt.core 4 969 2019-05-17 21:40:16.771
!MESSAGE on [in BasicStone [in [Working copy] BasicStone.java [in de.maltesermailo.tigmod.stones [in src/main/java [in MDKExample]]]]] does not exist

!ENTRY org.eclipse.jdt.ui 4 10001 2019-05-17 21:40:19.275
!MESSAGE Internal Error
!STACK 1
Java Model Exception: Java Model Status [onG [in BasicStone [in [Working copy] BasicStone.java [in de.maltesermailo.tigmod.stones [in src/main/java [in MDKExample]]]]] does not exist]
	at org.eclipse.jdt.internal.core.JavaElement.newNotPresentException(JavaElement.java:573)
	at org.eclipse.jdt.internal.core.JavaElement.openWhenClosed(JavaElement.java:607)
	at org.eclipse.jdt.internal.core.JavaElement.getElementInfo(JavaElement.java:326)
	at org.eclipse.jdt.internal.core.JavaElement.getElementInfo(JavaElement.java:312)
	at org.eclipse.jdt.internal.core.SourceRefElement.getSourceRange(SourceRefElement.java:233)
	at org.eclipse.jdt.internal.core.Member.getJavadocRange(Member.java:301)
	at org.eclipse.jdt.internal.ui.text.javadoc.JavadocContentAccess2.getHTMLContentFromSource(JavadocContentAccess2.java:664)
	at org.eclipse.jdt.internal.ui.text.javadoc.JavadocContentAccess2.getHTMLContent(JavadocContentAccess2.java:540)
	at org.eclipse.jdt.internal.ui.infoviews.JavadocView.getJavadocHtml(JavadocView.java:1109)
	at org.eclipse.jdt.internal.ui.infoviews.JavadocView.computeInput(JavadocView.java:894)
	at org.eclipse.jdt.internal.ui.infoviews.AbstractInfoView$2.run(AbstractInfoView.java:761)
!SUBENTRY 1 org.eclipse.jdt.core 4 969 2019-05-17 21:40:19.275
!MESSAGE onG [in BasicStone [in [Working copy] BasicStone.java [in de.maltesermailo.tigmod.stones [in src/main/java [in MDKExample]]]]] does not exist

!ENTRY org.eclipse.jdt.ui 4 10001 2019-05-17 21:40:19.279
!MESSAGE Internal Error
!STACK 1
Java Model Exception: Java Model Status [on [in BasicStone [in [Working copy] BasicStone.java [in de.maltesermailo.tigmod.stones [in src/main/java [in MDKExample]]]]] does not exist]
	at org.eclipse.jdt.internal.core.JavaElement.newNotPresentException(JavaElement.java:573)
	at org.eclipse.jdt.internal.core.JavaElement.openWhenClosed(JavaElement.java:607)
	at org.eclipse.jdt.internal.core.JavaElement.getElementInfo(JavaElement.java:326)
	at org.eclipse.jdt.internal.core.JavaElement.getElementInfo(JavaElement.java:312)
	at org.eclipse.jdt.internal.core.SourceRefElement.getSourceRange(SourceRefElement.java:233)
	at org.eclipse.jdt.internal.core.Member.getJavadocRange(Member.java:301)
	at org.eclipse.jdt.internal.ui.text.javadoc.JavadocContentAccess2.getHTMLContentFromSource(JavadocContentAccess2.java:664)
	at org.eclipse.jdt.internal.ui.text.javadoc.JavadocContentAccess2.getHTMLContent(JavadocContentAccess2.java:540)
	at org.eclipse.jdt.internal.ui.infoviews.JavadocView.getJavadocHtml(JavadocView.java:1109)
	at org.eclipse.jdt.internal.ui.infoviews.JavadocView.computeInput(JavadocView.java:894)
	at org.eclipse.jdt.internal.ui.infoviews.AbstractInfoView$2.run(AbstractInfoView.java:761)
!SUBENTRY 1 org.eclipse.jdt.core 4 969 2019-05-17 21:40:19.279
!MESSAGE on [in BasicStone [in [Working copy] BasicStone.java [in de.maltesermailo.tigmod.stones [in src/main/java [in MDKExample]]]]] does not exist

!ENTRY org.eclipse.jdt.ui 4 10001 2019-05-17 21:52:59.956
!MESSAGE Internal Error
!STACK 1
Java Model Exception: Java Model Status [changeMode(ItemStack, EntityPlayer) [in MindStone [in [Working copy] MindStone.java [in de.maltesermailo.tigmod.stones [in src/main/java [in MDKExample]]]]] does not exist]
	at org.eclipse.jdt.internal.core.JavaElement.newNotPresentException(JavaElement.java:573)
	at org.eclipse.jdt.internal.core.JavaElement.openWhenClosed(JavaElement.java:607)
	at org.eclipse.jdt.internal.core.JavaElement.getElementInfo(JavaElement.java:326)
	at org.eclipse.jdt.internal.core.JavaElement.getElementInfo(JavaElement.java:312)
	at org.eclipse.jdt.internal.core.SourceRefElement.getSourceRange(SourceRefElement.java:233)
	at org.eclipse.jdt.internal.core.Member.getJavadocRange(Member.java:301)
	at org.eclipse.jdt.internal.ui.text.javadoc.JavadocContentAccess2.getHTMLContentFromSource(JavadocContentAccess2.java:664)
	at org.eclipse.jdt.internal.ui.text.javadoc.JavadocContentAccess2.getHTMLContent(JavadocContentAccess2.java:540)
	at org.eclipse.jdt.internal.ui.infoviews.JavadocView.getJavadocHtml(JavadocView.java:1109)
	at org.eclipse.jdt.internal.ui.infoviews.JavadocView.computeInput(JavadocView.java:894)
	at org.eclipse.jdt.internal.ui.infoviews.AbstractInfoView$2.run(AbstractInfoView.java:761)
!SUBENTRY 1 org.eclipse.jdt.core 4 969 2019-05-17 21:52:59.956
!MESSAGE changeMode(ItemStack, EntityPlayer) [in MindStone [in [Working copy] MindStone.java [in de.maltesermailo.tigmod.stones [in src/main/java [in MDKExample]]]]] does not exist

!ENTRY org.eclipse.jdt.ui 4 10001 2019-05-17 21:55:19.499
!MESSAGE Internal Error
!STACK 1
Java Model Exception: Java Model Status [changeMode(ItemStack, EntityPlayer) [in SoulStone [in [Working copy] SoulStone.java [in de.maltesermailo.tigmod.stones [in src/main/java [in MDKExample]]]]] does not exist]
	at org.eclipse.jdt.internal.core.JavaElement.newNotPresentException(JavaElement.java:573)
	at org.eclipse.jdt.internal.core.JavaElement.openWhenClosed(JavaElement.java:607)
	at org.eclipse.jdt.internal.core.JavaElement.getElementInfo(JavaElement.java:326)
	at org.eclipse.jdt.internal.core.JavaElement.getElementInfo(JavaElement.java:312)
	at org.eclipse.jdt.internal.core.SourceRefElement.getSourceRange(SourceRefElement.java:233)
	at org.eclipse.jdt.internal.core.Member.getJavadocRange(Member.java:301)
	at org.eclipse.jdt.internal.ui.text.javadoc.JavadocContentAccess2.getHTMLContentFromSource(JavadocContentAccess2.java:664)
	at org.eclipse.jdt.internal.ui.text.javadoc.JavadocContentAccess2.getHTMLContent(JavadocContentAccess2.java:540)
	at org.eclipse.jdt.internal.ui.infoviews.JavadocView.getJavadocHtml(JavadocView.java:1109)
	at org.eclipse.jdt.internal.ui.infoviews.JavadocView.computeInput(JavadocView.java:894)
	at org.eclipse.jdt.internal.ui.infoviews.AbstractInfoView$2.run(AbstractInfoView.java:761)
!SUBENTRY 1 org.eclipse.jdt.core 4 969 2019-05-17 21:55:19.499
!MESSAGE changeMode(ItemStack, EntityPlayer) [in SoulStone [in [Working copy] SoulStone.java [in de.maltesermailo.tigmod.stones [in src/main/java [in MDKExample]]]]] does not exist

!ENTRY org.eclipse.jdt.core.manipulation 4 10001 2019-05-17 22:07:50.762
!MESSAGE 
!STACK 1
Java Model Exception: Java Model Status [chasAStoneAtAll(ItemStack) [in InfinityGauntlet [in [Working copy] InfinityGauntlet.java [in de.maltesermailo.tigmod.gauntlet [in src/main/java [in MDKExample]]]]] does not exist]
	at org.eclipse.jdt.internal.core.JavaElement.newNotPresentException(JavaElement.java:573)
	at org.eclipse.jdt.internal.core.JavaElement.openWhenClosed(JavaElement.java:607)
	at org.eclipse.jdt.internal.core.JavaElement.getElementInfo(JavaElement.java:326)
	at org.eclipse.jdt.internal.core.JavaElement.getElementInfo(JavaElement.java:312)
	at org.eclipse.jdt.internal.core.SourceMethod.getParameters(SourceMethod.java:153)
	at org.eclipse.jdt.internal.core.manipulation.JavaElementLabelComposerCore.appendMethodLabel(JavaElementLabelComposerCore.java:345)
	at org.eclipse.jdt.internal.core.manipulation.JavaElementLabelComposerCore.appendElementLabel(JavaElementLabelComposerCore.java:171)
	at org.eclipse.jdt.internal.ui.viewsupport.JavaElementLinks.getElementLabel(JavaElementLinks.java:709)
	at org.eclipse.jdt.internal.ui.viewsupport.JavaElementLinks.getElementLabel(JavaElementLinks.java:691)
	at org.eclipse.jdt.internal.ui.text.java.hover.JavadocHover.getInfoText(JavadocHover.java:818)
	at org.eclipse.jdt.internal.ui.text.java.hover.JavadocHover.getHoverInfo(JavadocHover.java:755)
	at org.eclipse.jdt.internal.ui.text.java.hover.JavadocHover.internalGetHoverInfo(JavadocHover.java:678)
	at org.eclipse.jdt.internal.ui.text.java.hover.JavadocHover.getHoverInfo2(JavadocHover.java:670)
	at org.eclipse.jdt.internal.ui.text.java.hover.BestMatchHover.getHoverInfo2(BestMatchHover.java:167)
	at org.eclipse.jdt.internal.ui.text.java.hover.BestMatchHover.getHoverInfo2(BestMatchHover.java:133)
	at org.eclipse.jdt.internal.ui.text.java.hover.JavaEditorTextHoverProxy.getHoverInfo2(JavaEditorTextHoverProxy.java:89)
	at org.eclipse.jface.text.TextViewerHoverManager$4.run(TextViewerHoverManager.java:169)
!SUBENTRY 1 org.eclipse.jdt.core 4 969 2019-05-17 22:07:50.762
!MESSAGE chasAStoneAtAll(ItemStack) [in InfinityGauntlet [in [Working copy] InfinityGauntlet.java [in de.maltesermailo.tigmod.gauntlet [in src/main/java [in MDKExample]]]]] does not exist

!ENTRY org.eclipse.jdt.ui 4 10001 2019-05-17 22:07:50.766
!MESSAGE Internal Error
!STACK 1
Java Model Exception: Java Model Status [chasAStoneAtAll(ItemStack) [in InfinityGauntlet [in [Working copy] InfinityGauntlet.java [in de.maltesermailo.tigmod.gauntlet [in src/main/java [in MDKExample]]]]] does not exist]
	at org.eclipse.jdt.internal.core.JavaElement.newNotPresentException(JavaElement.java:573)
	at org.eclipse.jdt.internal.core.JavaElement.openWhenClosed(JavaElement.java:607)
	at org.eclipse.jdt.internal.core.JavaElement.getElementInfo(JavaElement.java:326)
	at org.eclipse.jdt.internal.core.JavaElement.getElementInfo(JavaElement.java:312)
	at org.eclipse.jdt.internal.core.SourceRefElement.getSourceRange(SourceRefElement.java:233)
	at org.eclipse.jdt.internal.core.Member.getJavadocRange(Member.java:301)
	at org.eclipse.jdt.internal.ui.text.javadoc.JavadocContentAccess2.getHTMLContentFromSource(JavadocContentAccess2.java:664)
	at org.eclipse.jdt.internal.ui.text.javadoc.JavadocContentAccess2.getHTMLContent(JavadocContentAccess2.java:540)
	at org.eclipse.jdt.internal.ui.text.java.hover.JavadocHover.getHoverInfo(JavadocHover.java:760)
	at org.eclipse.jdt.internal.ui.text.java.hover.JavadocHover.internalGetHoverInfo(JavadocHover.java:678)
	at org.eclipse.jdt.internal.ui.text.java.hover.JavadocHover.getHoverInfo2(JavadocHover.java:670)
	at org.eclipse.jdt.internal.ui.text.java.hover.BestMatchHover.getHoverInfo2(BestMatchHover.java:167)
	at org.eclipse.jdt.internal.ui.text.java.hover.BestMatchHover.getHoverInfo2(BestMatchHover.java:133)
	at org.eclipse.jdt.internal.ui.text.java.hover.JavaEditorTextHoverProxy.getHoverInfo2(JavaEditorTextHoverProxy.java:89)
	at org.eclipse.jface.text.TextViewerHoverManager$4.run(TextViewerHoverManager.java:169)
!SUBENTRY 1 org.eclipse.jdt.core 4 969 2019-05-17 22:07:50.766
!MESSAGE chasAStoneAtAll(ItemStack) [in InfinityGauntlet [in [Working copy] InfinityGauntlet.java [in de.maltesermailo.tigmod.gauntlet [in src/main/java [in MDKExample]]]]] does not exist

!ENTRY org.eclipse.jdt.ui 4 10001 2019-05-17 22:13:27.293
!MESSAGE Internal Error
!STACK 1
Java Model Exception: Java Model Status [onEntityInteract(BlockInt) [in TIGMod [in [Working copy] TIGMod.java [in de.maltesermailo.tigmod [in src/main/java [in MDKExample]]]]] does not exist]
	at org.eclipse.jdt.internal.core.JavaElement.newNotPresentException(JavaElement.java:573)
	at org.eclipse.jdt.internal.core.JavaElement.openWhenClosed(JavaElement.java:607)
	at org.eclipse.jdt.internal.core.JavaElement.getElementInfo(JavaElement.java:326)
	at org.eclipse.jdt.internal.core.JavaElement.getElementInfo(JavaElement.java:312)
	at org.eclipse.jdt.internal.core.SourceRefElement.getSourceRange(SourceRefElement.java:233)
	at org.eclipse.jdt.internal.core.Member.getJavadocRange(Member.java:301)
	at org.eclipse.jdt.internal.ui.text.javadoc.JavadocContentAccess2.getHTMLContentFromSource(JavadocContentAccess2.java:664)
	at org.eclipse.jdt.internal.ui.text.javadoc.JavadocContentAccess2.getHTMLContent(JavadocContentAccess2.java:540)
	at org.eclipse.jdt.internal.ui.infoviews.JavadocView.getJavadocHtml(JavadocView.java:1109)
	at org.eclipse.jdt.internal.ui.infoviews.JavadocView.computeInput(JavadocView.java:894)
	at org.eclipse.jdt.internal.ui.infoviews.AbstractInfoView$2.run(AbstractInfoView.java:761)
!SUBENTRY 1 org.eclipse.jdt.core 4 969 2019-05-17 22:13:27.293
!MESSAGE onEntityInteract(BlockInt) [in TIGMod [in [Working copy] TIGMod.java [in de.maltesermailo.tigmod [in src/main/java [in MDKExample]]]]] does not exist

!ENTRY org.eclipse.jdt.ui 4 10001 2019-05-18 00:58:19.805
!MESSAGE Internal Error
!STACK 1
Java Model Exception: Java Model Status [changeMode(ItemStack, EntityPlayer, boolean) [in BasicStone [in [Working copy] BasicStone.java [in de.maltesermailo.tigmod.stones [in src/main/java [in MDKExample]]]]] does not exist]
	at org.eclipse.jdt.internal.core.JavaElement.newNotPresentException(JavaElement.java:573)
	at org.eclipse.jdt.internal.core.JavaElement.openWhenClosed(JavaElement.java:607)
	at org.eclipse.jdt.internal.core.JavaElement.getElementInfo(JavaElement.java:326)
	at org.eclipse.jdt.internal.core.JavaElement.getElementInfo(JavaElement.java:312)
	at org.eclipse.jdt.internal.core.SourceRefElement.getSourceRange(SourceRefElement.java:233)
	at org.eclipse.jdt.internal.core.Member.getJavadocRange(Member.java:301)
	at org.eclipse.jdt.internal.ui.text.javadoc.JavadocContentAccess2.getHTMLContentFromSource(JavadocContentAccess2.java:664)
	at org.eclipse.jdt.internal.ui.text.javadoc.JavadocContentAccess2.getHTMLContent(JavadocContentAccess2.java:540)
	at org.eclipse.jdt.internal.ui.text.java.hover.JavadocHover.getHoverInfo(JavadocHover.java:760)
	at org.eclipse.jdt.internal.ui.text.java.hover.JavadocHover.internalGetHoverInfo(JavadocHover.java:678)
	at org.eclipse.jdt.internal.ui.text.java.hover.JavadocHover.getHoverInfo2(JavadocHover.java:670)
	at org.eclipse.jdt.internal.ui.text.java.hover.BestMatchHover.getHoverInfo2(BestMatchHover.java:167)
	at org.eclipse.jdt.internal.ui.text.java.hover.BestMatchHover.getHoverInfo2(BestMatchHover.java:133)
	at org.eclipse.jdt.internal.ui.text.java.hover.JavaEditorTextHoverProxy.getHoverInfo2(JavaEditorTextHoverProxy.java:89)
	at org.eclipse.jface.text.TextViewerHoverManager$4.run(TextViewerHoverManager.java:169)
!SUBENTRY 1 org.eclipse.jdt.core 4 969 2019-05-18 00:58:19.805
!MESSAGE changeMode(ItemStack, EntityPlayer, boolean) [in BasicStone [in [Working copy] BasicStone.java [in de.maltesermailo.tigmod.stones [in src/main/java [in MDKExample]]]]] does not exist

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-18 01:43:49.199
!MESSAGE XMPP connection error:  (XMPPConnectionService.java:101)
!STACK 0
javax.net.ssl.SSLProtocolException: Connection reset
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:126)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:321)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:264)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:259)
	at java.base/sun.security.ssl.SSLSocketImpl.handleException(SSLSocketImpl.java:1314)
	at java.base/sun.security.ssl.SSLSocketImpl$AppInputStream.read(SSLSocketImpl.java:839)
	at java.base/sun.nio.cs.StreamDecoder.readBytes(StreamDecoder.java:284)
	at java.base/sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:326)
	at java.base/sun.nio.cs.StreamDecoder.read(StreamDecoder.java:178)
	at java.base/java.io.InputStreamReader.read(InputStreamReader.java:185)
	at java.base/java.io.BufferedReader.fill(BufferedReader.java:161)
	at java.base/java.io.BufferedReader.read1(BufferedReader.java:212)
	at java.base/java.io.BufferedReader.read(BufferedReader.java:287)
	at org.xmlpull.mxp1.MXParser.fillBuf(MXParser.java:2992)
	at org.xmlpull.mxp1.MXParser.more(MXParser.java:3046)
	at org.xmlpull.mxp1.MXParser.nextImpl(MXParser.java:1144)
	at org.xmlpull.mxp1.MXParser.next(MXParser.java:1093)
	at org.jivesoftware.smack.PacketReader.parsePackets(PacketReader.java:312)
	at org.jivesoftware.smack.PacketReader.access$000(PacketReader.java:48)
	at org.jivesoftware.smack.PacketReader$1.run(PacketReader.java:76)
Caused by: java.net.SocketException: Connection reset
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:186)
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:140)
	at java.base/sun.security.ssl.SSLSocketInputRecord.read(SSLSocketInputRecord.java:448)
	at java.base/sun.security.ssl.SSLSocketInputRecord.bytesInCompletePacket(SSLSocketInputRecord.java:68)
	at java.base/sun.security.ssl.SSLSocketImpl.readApplicationRecord(SSLSocketImpl.java:1104)
	at java.base/sun.security.ssl.SSLSocketImpl$AppInputStream.read(SSLSocketImpl.java:823)
	... 14 more

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-18 01:43:49.202
!MESSAGE  new connection state == ERROR (XMPPConnectionService.java:400)
!STACK 0
javax.net.ssl.SSLProtocolException: Connection reset
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:126)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:321)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:264)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:259)
	at java.base/sun.security.ssl.SSLSocketImpl.handleException(SSLSocketImpl.java:1314)
	at java.base/sun.security.ssl.SSLSocketImpl$AppInputStream.read(SSLSocketImpl.java:839)
	at java.base/sun.nio.cs.StreamDecoder.readBytes(StreamDecoder.java:284)
	at java.base/sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:326)
	at java.base/sun.nio.cs.StreamDecoder.read(StreamDecoder.java:178)
	at java.base/java.io.InputStreamReader.read(InputStreamReader.java:185)
	at java.base/java.io.BufferedReader.fill(BufferedReader.java:161)
	at java.base/java.io.BufferedReader.read1(BufferedReader.java:212)
	at java.base/java.io.BufferedReader.read(BufferedReader.java:287)
	at org.xmlpull.mxp1.MXParser.fillBuf(MXParser.java:2992)
	at org.xmlpull.mxp1.MXParser.more(MXParser.java:3046)
	at org.xmlpull.mxp1.MXParser.nextImpl(MXParser.java:1144)
	at org.xmlpull.mxp1.MXParser.next(MXParser.java:1093)
	at org.jivesoftware.smack.PacketReader.parsePackets(PacketReader.java:312)
	at org.jivesoftware.smack.PacketReader.access$000(PacketReader.java:48)
	at org.jivesoftware.smack.PacketReader$1.run(PacketReader.java:76)
Caused by: java.net.SocketException: Connection reset
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:186)
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:140)
	at java.base/sun.security.ssl.SSLSocketInputRecord.read(SSLSocketInputRecord.java:448)
	at java.base/sun.security.ssl.SSLSocketInputRecord.bytesInCompletePacket(SSLSocketInputRecord.java:68)
	at java.base/sun.security.ssl.SSLSocketImpl.readApplicationRecord(SSLSocketImpl.java:1104)
	at java.base/sun.security.ssl.SSLSocketImpl$AppInputStream.read(SSLSocketImpl.java:823)
	... 14 more

!ENTRY org.eclipse.ui 4 0 2019-05-18 01:43:51.857
!MESSAGE Unhandled event loop exception
!STACK 0
org.eclipse.swt.SWTException: Widget is disposed
	at org.eclipse.swt.SWT.error(SWT.java:4699)
	at org.eclipse.swt.SWT.error(SWT.java:4614)
	at org.eclipse.swt.SWT.error(SWT.java:4585)
	at org.eclipse.swt.widgets.Widget.error(Widget.java:451)
	at org.eclipse.swt.widgets.Widget.checkWidget(Widget.java:350)
	at org.eclipse.swt.widgets.Composite.getChildren(Composite.java:469)
	at org.eclipse.epp.internal.mpc.ui.wizards.MarketplaceDropAdapter.lambda$0(MarketplaceDropAdapter.java:136)
	at org.eclipse.swt.widgets.RunnableLock.run(RunnableLock.java:40)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:185)
	at org.eclipse.swt.widgets.Display.runAsyncMessages(Display.java:3919)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3550)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1173)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1062)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:155)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:644)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:566)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:150)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:155)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:203)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:137)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:107)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:400)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:255)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:661)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:597)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1476)

!ENTRY org.eclipse.core.resources 4 566 2019-05-18 03:15:37.114
!MESSAGE Workspace was not properly initialized or has already shutdown.

!ENTRY org.eclipse.core.jobs 4 1 2019-05-18 03:15:37.118
!MESSAGE Worker thread ended job: Refreshing workspace(10), but still holds rule: ThreadJob(Refreshing workspace(10),[F/MDKExample/run/saves/Copy of New World])
!SESSION 2019-05-18 17:40:19.767 -----------------------------------------------
eclipse.buildId=4.11.0.I20190307-0500
java.version=11.0.2
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=de_DE
Framework arguments:  -product org.eclipse.epp.package.java.product
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.java.product

!ENTRY org.eclipse.egit.core 1 0 2019-05-18 17:40:28.654
!MESSAGE Using Apache MINA sshd as ssh client.

!ENTRY org.eclipse.egit.ui 2 0 2019-05-18 17:40:41.557
!MESSAGE Warning: The environment variable HOME is not set. The following directory will be used to store the Git
user global configuration and to define the default location to store repositories: 'C:\Users\Jannik'. If this is
not correct please set the HOME environment variable and restart Eclipse. Otherwise Git for Windows and
EGit might behave differently since they see different configuration options.
This warning can be switched off on the Team > Git > Confirmations and Warnings preference page.

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-18 17:40:44.326
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-18 17:40:44.333
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-18 17:40:44.381
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-18 17:40:54.322
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.38.1:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-18 17:40:54.340
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.178.20:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-18 17:40:54.352
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.77.1:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-18 17:40:54.358
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /169.254.181.9:0 (StunServiceImpl.java:420)

!ENTRY org.eclipse.jdt.ui 4 10001 2019-05-18 20:46:09.718
!MESSAGE Internal Error
!STACK 1
Java Model Exception: Java Model Status [itemCrafted(ItemCraftedEventEvent) [in TIGMod [in [Working copy] TIGMod.java [in de.maltesermailo.tigmod [in src/main/java [in MDKExample]]]]] does not exist]
	at org.eclipse.jdt.internal.core.JavaElement.newNotPresentException(JavaElement.java:573)
	at org.eclipse.jdt.internal.core.JavaElement.openWhenClosed(JavaElement.java:607)
	at org.eclipse.jdt.internal.core.JavaElement.getElementInfo(JavaElement.java:326)
	at org.eclipse.jdt.internal.core.JavaElement.getElementInfo(JavaElement.java:312)
	at org.eclipse.jdt.internal.core.SourceRefElement.getSourceRange(SourceRefElement.java:233)
	at org.eclipse.jdt.internal.core.Member.getJavadocRange(Member.java:301)
	at org.eclipse.jdt.internal.ui.text.javadoc.JavadocContentAccess2.getHTMLContentFromSource(JavadocContentAccess2.java:664)
	at org.eclipse.jdt.internal.ui.text.javadoc.JavadocContentAccess2.getHTMLContent(JavadocContentAccess2.java:540)
	at org.eclipse.jdt.internal.ui.infoviews.JavadocView.getJavadocHtml(JavadocView.java:1109)
	at org.eclipse.jdt.internal.ui.infoviews.JavadocView.computeInput(JavadocView.java:894)
	at org.eclipse.jdt.internal.ui.infoviews.AbstractInfoView$2.run(AbstractInfoView.java:761)
!SUBENTRY 1 org.eclipse.jdt.core 4 969 2019-05-18 20:46:09.722
!MESSAGE itemCrafted(ItemCraftedEventEvent) [in TIGMod [in [Working copy] TIGMod.java [in de.maltesermailo.tigmod [in src/main/java [in MDKExample]]]]] does not exist

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-19 01:42:26.006
!MESSAGE XMPP connection error:  (XMPPConnectionService.java:101)
!STACK 0
javax.net.ssl.SSLProtocolException: Connection reset
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:126)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:321)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:264)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:259)
	at java.base/sun.security.ssl.SSLSocketImpl.handleException(SSLSocketImpl.java:1314)
	at java.base/sun.security.ssl.SSLSocketImpl$AppInputStream.read(SSLSocketImpl.java:839)
	at java.base/sun.nio.cs.StreamDecoder.readBytes(StreamDecoder.java:284)
	at java.base/sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:326)
	at java.base/sun.nio.cs.StreamDecoder.read(StreamDecoder.java:178)
	at java.base/java.io.InputStreamReader.read(InputStreamReader.java:185)
	at java.base/java.io.BufferedReader.fill(BufferedReader.java:161)
	at java.base/java.io.BufferedReader.read1(BufferedReader.java:212)
	at java.base/java.io.BufferedReader.read(BufferedReader.java:287)
	at org.xmlpull.mxp1.MXParser.fillBuf(MXParser.java:2992)
	at org.xmlpull.mxp1.MXParser.more(MXParser.java:3046)
	at org.xmlpull.mxp1.MXParser.nextImpl(MXParser.java:1144)
	at org.xmlpull.mxp1.MXParser.next(MXParser.java:1093)
	at org.jivesoftware.smack.PacketReader.parsePackets(PacketReader.java:312)
	at org.jivesoftware.smack.PacketReader.access$000(PacketReader.java:48)
	at org.jivesoftware.smack.PacketReader$1.run(PacketReader.java:76)
Caused by: java.net.SocketException: Connection reset
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:186)
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:140)
	at java.base/sun.security.ssl.SSLSocketInputRecord.read(SSLSocketInputRecord.java:448)
	at java.base/sun.security.ssl.SSLSocketInputRecord.bytesInCompletePacket(SSLSocketInputRecord.java:68)
	at java.base/sun.security.ssl.SSLSocketImpl.readApplicationRecord(SSLSocketImpl.java:1104)
	at java.base/sun.security.ssl.SSLSocketImpl$AppInputStream.read(SSLSocketImpl.java:823)
	... 14 more

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-19 01:42:26.065
!MESSAGE  new connection state == ERROR (XMPPConnectionService.java:400)
!STACK 0
javax.net.ssl.SSLProtocolException: Connection reset
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:126)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:321)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:264)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:259)
	at java.base/sun.security.ssl.SSLSocketImpl.handleException(SSLSocketImpl.java:1314)
	at java.base/sun.security.ssl.SSLSocketImpl$AppInputStream.read(SSLSocketImpl.java:839)
	at java.base/sun.nio.cs.StreamDecoder.readBytes(StreamDecoder.java:284)
	at java.base/sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:326)
	at java.base/sun.nio.cs.StreamDecoder.read(StreamDecoder.java:178)
	at java.base/java.io.InputStreamReader.read(InputStreamReader.java:185)
	at java.base/java.io.BufferedReader.fill(BufferedReader.java:161)
	at java.base/java.io.BufferedReader.read1(BufferedReader.java:212)
	at java.base/java.io.BufferedReader.read(BufferedReader.java:287)
	at org.xmlpull.mxp1.MXParser.fillBuf(MXParser.java:2992)
	at org.xmlpull.mxp1.MXParser.more(MXParser.java:3046)
	at org.xmlpull.mxp1.MXParser.nextImpl(MXParser.java:1144)
	at org.xmlpull.mxp1.MXParser.next(MXParser.java:1093)
	at org.jivesoftware.smack.PacketReader.parsePackets(PacketReader.java:312)
	at org.jivesoftware.smack.PacketReader.access$000(PacketReader.java:48)
	at org.jivesoftware.smack.PacketReader$1.run(PacketReader.java:76)
Caused by: java.net.SocketException: Connection reset
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:186)
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:140)
	at java.base/sun.security.ssl.SSLSocketInputRecord.read(SSLSocketInputRecord.java:448)
	at java.base/sun.security.ssl.SSLSocketInputRecord.bytesInCompletePacket(SSLSocketInputRecord.java:68)
	at java.base/sun.security.ssl.SSLSocketImpl.readApplicationRecord(SSLSocketImpl.java:1104)
	at java.base/sun.security.ssl.SSLSocketImpl$AppInputStream.read(SSLSocketImpl.java:823)
	... 14 more

!ENTRY org.eclipse.core.jobs 4 2 2019-05-19 03:03:28.511
!MESSAGE An internal error occurred during: "Requesting Java AST from selection".
!STACK 0
java.lang.ArrayIndexOutOfBoundsException: Index 23 out of bounds for length 14
	at org.eclipse.jdt.internal.compiler.util.HashtableOfPackage.get(HashtableOfPackage.java:59)
	at org.eclipse.jdt.internal.compiler.lookup.LookupEnvironment.getPackage0(LookupEnvironment.java:1594)
	at org.eclipse.jdt.internal.compiler.lookup.LookupEnvironment.getTopLevelPackage(LookupEnvironment.java:1627)
	at org.eclipse.jdt.internal.compiler.lookup.CompilationUnitScope.findImport(CompilationUnitScope.java:534)
	at org.eclipse.jdt.internal.compiler.lookup.CompilationUnitScope.getImport(CompilationUnitScope.java:687)
	at org.eclipse.jdt.core.dom.DefaultBindingResolver.resolveName(DefaultBindingResolver.java:1358)
	at org.eclipse.jdt.core.dom.Name.resolveBinding(Name.java:90)
	at org.eclipse.jdt.internal.core.manipulation.search.OccurrencesFinder.visit(OccurrencesFinder.java:149)
	at org.eclipse.jdt.core.dom.QualifiedName.accept0(QualifiedName.java:159)
	at org.eclipse.jdt.core.dom.ASTNode.accept(ASTNode.java:2815)
	at org.eclipse.jdt.core.dom.ASTNode.acceptChild(ASTNode.java:2863)
	at org.eclipse.jdt.core.dom.QualifiedName.accept0(QualifiedName.java:162)
	at org.eclipse.jdt.core.dom.ASTNode.accept(ASTNode.java:2815)
	at org.eclipse.jdt.core.dom.ASTNode.acceptChild(ASTNode.java:2863)
	at org.eclipse.jdt.core.dom.ImportDeclaration.accept0(ImportDeclaration.java:206)
	at org.eclipse.jdt.core.dom.ASTNode.accept(ASTNode.java:2815)
	at org.eclipse.jdt.core.dom.ASTNode.acceptChildren(ASTNode.java:2886)
	at org.eclipse.jdt.core.dom.CompilationUnit.accept0(CompilationUnit.java:257)
	at org.eclipse.jdt.core.dom.ASTNode.accept(ASTNode.java:2815)
	at org.eclipse.jdt.internal.core.manipulation.search.OccurrencesFinder.performSearch(OccurrencesFinder.java:104)
	at org.eclipse.jdt.internal.core.manipulation.search.OccurrencesFinder.getOccurrences(OccurrencesFinder.java:110)
	at org.eclipse.jdt.internal.ui.javaeditor.JavaEditor.updateOccurrenceAnnotations(JavaEditor.java:3371)
	at org.eclipse.jdt.internal.ui.javaeditor.JavaEditor$6.selectionChanged(JavaEditor.java:3397)
	at org.eclipse.jdt.internal.ui.viewsupport.SelectionListenerWithASTManager$PartListenerGroup.calculateASTandInform(SelectionListenerWithASTManager.java:184)
	at org.eclipse.jdt.internal.ui.viewsupport.SelectionListenerWithASTManager$PartListenerGroup$3.run(SelectionListenerWithASTManager.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.core.resources 4 566 2019-05-19 03:43:31.316
!MESSAGE Workspace was not properly initialized or has already shutdown.

!ENTRY org.eclipse.core.jobs 4 1 2019-05-19 03:43:31.323
!MESSAGE Worker thread ended job: Refreshing workspace(10), but still holds rule: ThreadJob(Refreshing workspace(10),[F/MDKExample/build/tmp/recompileMc/compiled/paulscode])
!SESSION 2019-05-19 22:06:44.179 -----------------------------------------------
eclipse.buildId=4.11.0.I20190307-0500
java.version=11.0.2
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=de_DE
Framework arguments:  -product org.eclipse.epp.package.java.product
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.java.product

!ENTRY org.eclipse.egit.core 1 0 2019-05-19 22:07:00.441
!MESSAGE Using Apache MINA sshd as ssh client.

!ENTRY org.eclipse.egit.ui 2 0 2019-05-19 22:07:12.656
!MESSAGE Warning: The environment variable HOME is not set. The following directory will be used to store the Git
user global configuration and to define the default location to store repositories: 'C:\Users\Jannik'. If this is
not correct please set the HOME environment variable and restart Eclipse. Otherwise Git for Windows and
EGit might behave differently since they see different configuration options.
This warning can be switched off on the Team > Git > Confirmations and Warnings preference page.

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-19 22:07:16.981
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-19 22:07:16.987
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-19 22:07:16.990
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-19 22:07:26.978
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.77.1:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-19 22:07:26.982
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /169.254.181.9:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-19 22:07:26.986
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.178.20:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-19 22:07:26.989
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.38.1:0 (StunServiceImpl.java:420)

!ENTRY org.eclipse.core.resources 4 566 2019-05-19 22:20:46.564
!MESSAGE Workspace was not properly initialized or has already shutdown.

!ENTRY org.eclipse.core.jobs 4 1 2019-05-19 22:20:46.567
!MESSAGE Worker thread ended job: Refreshing workspace(10), but still holds rule: ThreadJob(Refreshing workspace(10),[F/MDKExample/eclipse/.metadata/.plugins/org.eclipse.core.resources/.history])
!SESSION 2019-05-20 19:25:58.428 -----------------------------------------------
eclipse.buildId=4.11.0.I20190307-0500
java.version=11.0.2
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=de_DE
Framework arguments:  -product org.eclipse.epp.package.java.product
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.java.product

!ENTRY org.eclipse.egit.core 1 0 2019-05-20 19:26:05.843
!MESSAGE Using Apache MINA sshd as ssh client.

!ENTRY org.eclipse.egit.ui 2 0 2019-05-20 19:26:15.071
!MESSAGE Warning: The environment variable HOME is not set. The following directory will be used to store the Git
user global configuration and to define the default location to store repositories: 'C:\Users\Jannik'. If this is
not correct please set the HOME environment variable and restart Eclipse. Otherwise Git for Windows and
EGit might behave differently since they see different configuration options.
This warning can be switched off on the Team > Git > Confirmations and Warnings preference page.

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-20 19:26:18.846
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-20 19:26:18.865
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-20 19:26:18.896
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-20 19:26:28.843
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.178.20:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-20 19:26:28.847
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /169.254.181.9:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-20 19:26:28.849
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.77.1:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-20 19:26:28.852
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.38.1:0 (StunServiceImpl.java:420)

!ENTRY org.eclipse.core.resources 4 566 2019-05-20 22:14:36.026
!MESSAGE Workspace was not properly initialized or has already shutdown.
!SESSION 2019-05-21 17:23:49.264 -----------------------------------------------
eclipse.buildId=4.11.0.I20190307-0500
java.version=11.0.2
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=de_DE
Framework arguments:  -product org.eclipse.epp.package.java.product
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.java.product

!ENTRY org.eclipse.egit.core 1 0 2019-05-21 17:23:56.821
!MESSAGE Using Apache MINA sshd as ssh client.

!ENTRY org.eclipse.egit.ui 2 0 2019-05-21 17:24:12.784
!MESSAGE Warning: The environment variable HOME is not set. The following directory will be used to store the Git
user global configuration and to define the default location to store repositories: 'C:\Users\Jannik'. If this is
not correct please set the HOME environment variable and restart Eclipse. Otherwise Git for Windows and
EGit might behave differently since they see different configuration options.
This warning can be switched off on the Team > Git > Confirmations and Warnings preference page.

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-21 17:24:17.885
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-21 17:24:17.953
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-21 17:24:17.983
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-21 17:24:27.845
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.38.1:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-21 17:24:27.863
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /169.254.181.9:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-21 17:24:27.868
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.77.1:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-21 17:24:27.870
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.178.20:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-21 22:20:13.370
!MESSAGE XMPP connection error:  (XMPPConnectionService.java:101)
!STACK 0
javax.net.ssl.SSLProtocolException: Software caused connection abort: recv failed
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:126)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:321)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:264)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:259)
	at java.base/sun.security.ssl.SSLSocketImpl.handleException(SSLSocketImpl.java:1314)
	at java.base/sun.security.ssl.SSLSocketImpl$AppInputStream.read(SSLSocketImpl.java:839)
	at java.base/sun.nio.cs.StreamDecoder.readBytes(StreamDecoder.java:284)
	at java.base/sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:326)
	at java.base/sun.nio.cs.StreamDecoder.read(StreamDecoder.java:178)
	at java.base/java.io.InputStreamReader.read(InputStreamReader.java:185)
	at java.base/java.io.BufferedReader.fill(BufferedReader.java:161)
	at java.base/java.io.BufferedReader.read1(BufferedReader.java:212)
	at java.base/java.io.BufferedReader.read(BufferedReader.java:287)
	at org.xmlpull.mxp1.MXParser.fillBuf(MXParser.java:2992)
	at org.xmlpull.mxp1.MXParser.more(MXParser.java:3046)
	at org.xmlpull.mxp1.MXParser.nextImpl(MXParser.java:1144)
	at org.xmlpull.mxp1.MXParser.next(MXParser.java:1093)
	at org.jivesoftware.smack.PacketReader.parsePackets(PacketReader.java:312)
	at org.jivesoftware.smack.PacketReader.access$000(PacketReader.java:48)
	at org.jivesoftware.smack.PacketReader$1.run(PacketReader.java:76)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.base/java.net.SocketInputStream.socketRead0(Native Method)
	at java.base/java.net.SocketInputStream.socketRead(SocketInputStream.java:115)
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:168)
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:140)
	at java.base/sun.security.ssl.SSLSocketInputRecord.read(SSLSocketInputRecord.java:448)
	at java.base/sun.security.ssl.SSLSocketInputRecord.bytesInCompletePacket(SSLSocketInputRecord.java:68)
	at java.base/sun.security.ssl.SSLSocketImpl.readApplicationRecord(SSLSocketImpl.java:1104)
	at java.base/sun.security.ssl.SSLSocketImpl$AppInputStream.read(SSLSocketImpl.java:823)
	... 14 more

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-21 22:20:13.522
!MESSAGE  new connection state == ERROR (XMPPConnectionService.java:400)
!STACK 0
javax.net.ssl.SSLProtocolException: Software caused connection abort: recv failed
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:126)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:321)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:264)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:259)
	at java.base/sun.security.ssl.SSLSocketImpl.handleException(SSLSocketImpl.java:1314)
	at java.base/sun.security.ssl.SSLSocketImpl$AppInputStream.read(SSLSocketImpl.java:839)
	at java.base/sun.nio.cs.StreamDecoder.readBytes(StreamDecoder.java:284)
	at java.base/sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:326)
	at java.base/sun.nio.cs.StreamDecoder.read(StreamDecoder.java:178)
	at java.base/java.io.InputStreamReader.read(InputStreamReader.java:185)
	at java.base/java.io.BufferedReader.fill(BufferedReader.java:161)
	at java.base/java.io.BufferedReader.read1(BufferedReader.java:212)
	at java.base/java.io.BufferedReader.read(BufferedReader.java:287)
	at org.xmlpull.mxp1.MXParser.fillBuf(MXParser.java:2992)
	at org.xmlpull.mxp1.MXParser.more(MXParser.java:3046)
	at org.xmlpull.mxp1.MXParser.nextImpl(MXParser.java:1144)
	at org.xmlpull.mxp1.MXParser.next(MXParser.java:1093)
	at org.jivesoftware.smack.PacketReader.parsePackets(PacketReader.java:312)
	at org.jivesoftware.smack.PacketReader.access$000(PacketReader.java:48)
	at org.jivesoftware.smack.PacketReader$1.run(PacketReader.java:76)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.base/java.net.SocketInputStream.socketRead0(Native Method)
	at java.base/java.net.SocketInputStream.socketRead(SocketInputStream.java:115)
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:168)
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:140)
	at java.base/sun.security.ssl.SSLSocketInputRecord.read(SSLSocketInputRecord.java:448)
	at java.base/sun.security.ssl.SSLSocketInputRecord.bytesInCompletePacket(SSLSocketInputRecord.java:68)
	at java.base/sun.security.ssl.SSLSocketImpl.readApplicationRecord(SSLSocketImpl.java:1104)
	at java.base/sun.security.ssl.SSLSocketImpl$AppInputStream.read(SSLSocketImpl.java:823)
	... 14 more

!ENTRY org.eclipse.core.resources 4 566 2019-05-21 22:48:00.147
!MESSAGE Workspace was not properly initialized or has already shutdown.

!ENTRY org.eclipse.core.jobs 4 1 2019-05-21 22:48:00.150
!MESSAGE Worker thread ended job: Refreshing workspace(10), but still holds rule: ThreadJob(Refreshing workspace(10),[F/MDKExample/build/tmp/recompileMc/compiled/META-INF])
!SESSION 2019-05-22 15:23:44.425 -----------------------------------------------
eclipse.buildId=4.11.0.I20190307-0500
java.version=11.0.2
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=de_DE
Framework arguments:  -product org.eclipse.epp.package.java.product
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.java.product

!ENTRY org.eclipse.egit.core 1 0 2019-05-22 15:23:52.989
!MESSAGE Using Apache MINA sshd as ssh client.

!ENTRY org.eclipse.egit.ui 2 0 2019-05-22 15:24:05.713
!MESSAGE Warning: The environment variable HOME is not set. The following directory will be used to store the Git
user global configuration and to define the default location to store repositories: 'C:\Users\Jannik'. If this is
not correct please set the HOME environment variable and restart Eclipse. Otherwise Git for Windows and
EGit might behave differently since they see different configuration options.
This warning can be switched off on the Team > Git > Confirmations and Warnings preference page.

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-22 15:24:09.997
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-22 15:24:10.002
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-22 15:24:10.006
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-22 15:24:13.617
!MESSAGE an error occurred while performing a STUN discovery: ICMP Port Unreachable (StunServiceImpl.java:155)
!STACK 0
java.net.PortUnreachableException: ICMP Port Unreachable
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketReceiveOrPeekData(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.receive0(DualStackPlainDatagramSocketImpl.java:124)
	at java.base/java.net.AbstractPlainDatagramSocketImpl.receive(AbstractPlainDatagramSocketImpl.java:181)
	at java.base/java.net.DatagramSocket.receive(DatagramSocket.java:814)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery.performStunDiscovery(StunServiceImpl.java:284)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery.run(StunServiceImpl.java:151)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-22 15:24:19.996
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /169.254.181.9:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-22 15:24:20.000
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.38.1:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-22 15:24:20.004
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.77.1:0 (StunServiceImpl.java:420)
!SESSION 2019-05-23 19:34:21.025 -----------------------------------------------
eclipse.buildId=4.11.0.I20190307-0500
java.version=11.0.2
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=de_DE
Framework arguments:  -product org.eclipse.epp.package.java.product
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.java.product

!ENTRY org.eclipse.egit.core 1 0 2019-05-23 19:34:29.709
!MESSAGE Using Apache MINA sshd as ssh client.

!ENTRY org.eclipse.egit.ui 2 0 2019-05-23 19:35:01.603
!MESSAGE Warning: The environment variable HOME is not set. The following directory will be used to store the Git
user global configuration and to define the default location to store repositories: 'C:\Users\Jannik'. If this is
not correct please set the HOME environment variable and restart Eclipse. Otherwise Git for Windows and
EGit might behave differently since they see different configuration options.
This warning can be switched off on the Team > Git > Confirmations and Warnings preference page.

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-23 19:35:06.513
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-23 19:35:06.945
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-23 19:35:07.039
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-23 19:35:21.667
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /169.254.181.9:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-23 19:35:21.672
!MESSAGE an error occurred while performing a STUN discovery: ICMP Port Unreachable (StunServiceImpl.java:155)
!STACK 0
java.net.PortUnreachableException: ICMP Port Unreachable
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketReceiveOrPeekData(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.receive0(DualStackPlainDatagramSocketImpl.java:124)
	at java.base/java.net.AbstractPlainDatagramSocketImpl.receive(AbstractPlainDatagramSocketImpl.java:181)
	at java.base/java.net.DatagramSocket.receive(DatagramSocket.java:814)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery.performStunDiscovery(StunServiceImpl.java:284)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery.run(StunServiceImpl.java:151)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-23 19:35:21.677
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.77.1:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-23 19:35:21.686
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.38.1:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-23 20:56:17.850
!MESSAGE XMPP connection error:  (XMPPConnectionService.java:101)
!STACK 0
javax.net.ssl.SSLProtocolException: Connection reset
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:126)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:321)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:264)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:259)
	at java.base/sun.security.ssl.SSLSocketImpl.handleException(SSLSocketImpl.java:1314)
	at java.base/sun.security.ssl.SSLSocketImpl$AppInputStream.read(SSLSocketImpl.java:839)
	at java.base/sun.nio.cs.StreamDecoder.readBytes(StreamDecoder.java:284)
	at java.base/sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:326)
	at java.base/sun.nio.cs.StreamDecoder.read(StreamDecoder.java:178)
	at java.base/java.io.InputStreamReader.read(InputStreamReader.java:185)
	at java.base/java.io.BufferedReader.fill(BufferedReader.java:161)
	at java.base/java.io.BufferedReader.read1(BufferedReader.java:212)
	at java.base/java.io.BufferedReader.read(BufferedReader.java:287)
	at org.xmlpull.mxp1.MXParser.fillBuf(MXParser.java:2992)
	at org.xmlpull.mxp1.MXParser.more(MXParser.java:3046)
	at org.xmlpull.mxp1.MXParser.nextImpl(MXParser.java:1144)
	at org.xmlpull.mxp1.MXParser.next(MXParser.java:1093)
	at org.jivesoftware.smack.PacketReader.parsePackets(PacketReader.java:312)
	at org.jivesoftware.smack.PacketReader.access$000(PacketReader.java:48)
	at org.jivesoftware.smack.PacketReader$1.run(PacketReader.java:76)
Caused by: java.net.SocketException: Connection reset
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:186)
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:140)
	at java.base/sun.security.ssl.SSLSocketInputRecord.read(SSLSocketInputRecord.java:448)
	at java.base/sun.security.ssl.SSLSocketInputRecord.bytesInCompletePacket(SSLSocketInputRecord.java:68)
	at java.base/sun.security.ssl.SSLSocketImpl.readApplicationRecord(SSLSocketImpl.java:1104)
	at java.base/sun.security.ssl.SSLSocketImpl$AppInputStream.read(SSLSocketImpl.java:823)
	... 14 more

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-23 20:56:17.942
!MESSAGE  new connection state == ERROR (XMPPConnectionService.java:400)
!STACK 0
javax.net.ssl.SSLProtocolException: Connection reset
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:126)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:321)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:264)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:259)
	at java.base/sun.security.ssl.SSLSocketImpl.handleException(SSLSocketImpl.java:1314)
	at java.base/sun.security.ssl.SSLSocketImpl$AppInputStream.read(SSLSocketImpl.java:839)
	at java.base/sun.nio.cs.StreamDecoder.readBytes(StreamDecoder.java:284)
	at java.base/sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:326)
	at java.base/sun.nio.cs.StreamDecoder.read(StreamDecoder.java:178)
	at java.base/java.io.InputStreamReader.read(InputStreamReader.java:185)
	at java.base/java.io.BufferedReader.fill(BufferedReader.java:161)
	at java.base/java.io.BufferedReader.read1(BufferedReader.java:212)
	at java.base/java.io.BufferedReader.read(BufferedReader.java:287)
	at org.xmlpull.mxp1.MXParser.fillBuf(MXParser.java:2992)
	at org.xmlpull.mxp1.MXParser.more(MXParser.java:3046)
	at org.xmlpull.mxp1.MXParser.nextImpl(MXParser.java:1144)
	at org.xmlpull.mxp1.MXParser.next(MXParser.java:1093)
	at org.jivesoftware.smack.PacketReader.parsePackets(PacketReader.java:312)
	at org.jivesoftware.smack.PacketReader.access$000(PacketReader.java:48)
	at org.jivesoftware.smack.PacketReader$1.run(PacketReader.java:76)
Caused by: java.net.SocketException: Connection reset
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:186)
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:140)
	at java.base/sun.security.ssl.SSLSocketInputRecord.read(SSLSocketInputRecord.java:448)
	at java.base/sun.security.ssl.SSLSocketInputRecord.bytesInCompletePacket(SSLSocketInputRecord.java:68)
	at java.base/sun.security.ssl.SSLSocketImpl.readApplicationRecord(SSLSocketImpl.java:1104)
	at java.base/sun.security.ssl.SSLSocketImpl$AppInputStream.read(SSLSocketImpl.java:823)
	... 14 more
!SESSION 2019-05-24 17:03:45.653 -----------------------------------------------
eclipse.buildId=4.11.0.I20190307-0500
java.version=11.0.2
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=de_DE
Framework arguments:  -product org.eclipse.epp.package.java.product
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.java.product

!ENTRY org.eclipse.egit.core 1 0 2019-05-24 17:03:52.436
!MESSAGE Using Apache MINA sshd as ssh client.

!ENTRY org.eclipse.egit.ui 2 0 2019-05-24 17:04:02.312
!MESSAGE Warning: The environment variable HOME is not set. The following directory will be used to store the Git
user global configuration and to define the default location to store repositories: 'C:\Users\Jannik'. If this is
not correct please set the HOME environment variable and restart Eclipse. Otherwise Git for Windows and
EGit might behave differently since they see different configuration options.
This warning can be switched off on the Team > Git > Confirmations and Warnings preference page.

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-24 17:04:06.858
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-24 17:04:06.862
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-24 17:04:06.865
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-24 17:04:16.856
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.77.1:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-24 17:04:16.868
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.178.20:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-24 17:04:16.873
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /169.254.181.9:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-24 17:04:16.878
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.38.1:0 (StunServiceImpl.java:420)

!ENTRY org.eclipse.core.resources 4 566 2019-05-24 23:34:28.028
!MESSAGE Workspace was not properly initialized or has already shutdown.
!SESSION 2019-05-25 15:07:04.872 -----------------------------------------------
eclipse.buildId=4.11.0.I20190307-0500
java.version=11.0.2
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=de_DE
Framework arguments:  -product org.eclipse.epp.package.java.product
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.java.product

!ENTRY org.eclipse.egit.core 1 0 2019-05-25 15:07:34.902
!MESSAGE Using Apache MINA sshd as ssh client.

!ENTRY org.eclipse.egit.ui 2 0 2019-05-25 15:08:08.483
!MESSAGE Warning: The environment variable HOME is not set. The following directory will be used to store the Git
user global configuration and to define the default location to store repositories: 'C:\Users\Jannik'. If this is
not correct please set the HOME environment variable and restart Eclipse. Otherwise Git for Windows and
EGit might behave differently since they see different configuration options.
This warning can be switched off on the Team > Git > Confirmations and Warnings preference page.

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-25 15:08:16.957
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-25 15:08:17.195
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-25 15:08:18.153
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-25 15:08:26.954
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.178.20:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-25 15:08:27.045
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /169.254.181.9:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-25 15:08:27.533
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.77.1:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-25 15:08:27.536
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.38.1:0 (StunServiceImpl.java:420)

!ENTRY org.eclipse.debug.core 4 120 2019-05-25 19:50:44.802
!MESSAGE Variable references empty selection: ${project_loc}

!ENTRY org.eclipse.core.resources 2 10035 2019-05-25 22:57:29.415
!MESSAGE The workspace will exit with unsaved changes in this session.

!ENTRY org.eclipse.core.resources 4 566 2019-05-25 22:57:29.542
!MESSAGE Workspace was not properly initialized or has already shutdown.

!ENTRY org.eclipse.core.jobs 4 1 2019-05-25 22:57:29.547
!MESSAGE Worker thread ended job: Refreshing workspace(10), but still holds rule: ThreadJob(Refreshing workspace(10),[F/MDKExample/eclipse/.metadata/.plugins])
!SESSION 2019-05-26 15:33:59.284 -----------------------------------------------
eclipse.buildId=4.11.0.I20190307-0500
java.version=11.0.2
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=de_DE
Framework arguments:  -product org.eclipse.epp.package.java.product
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.java.product

!ENTRY org.eclipse.core.resources 2 10035 2019-05-26 15:34:04.044
!MESSAGE The workspace exited with unsaved changes in the previous session; refreshing workspace to recover changes.

!ENTRY org.eclipse.egit.core 1 0 2019-05-26 15:34:07.902
!MESSAGE Using Apache MINA sshd as ssh client.

!ENTRY org.eclipse.egit.ui 2 0 2019-05-26 15:34:58.434
!MESSAGE Warning: The environment variable HOME is not set. The following directory will be used to store the Git
user global configuration and to define the default location to store repositories: 'C:\Users\Jannik'. If this is
not correct please set the HOME environment variable and restart Eclipse. Otherwise Git for Windows and
EGit might behave differently since they see different configuration options.
This warning can be switched off on the Team > Git > Confirmations and Warnings preference page.

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-26 15:35:05.004
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-26 15:35:05.084
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-26 15:35:05.152
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-26 15:35:14.992
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.38.1:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-26 15:35:15.029
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /169.254.181.9:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-26 15:35:15.037
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.77.1:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-26 15:35:15.040
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.178.20:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-26 22:14:56.251
!MESSAGE XMPP connection error:  (XMPPConnectionService.java:101)
!STACK 0
javax.net.ssl.SSLProtocolException: Software caused connection abort: recv failed
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:126)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:321)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:264)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:259)
	at java.base/sun.security.ssl.SSLSocketImpl.handleException(SSLSocketImpl.java:1314)
	at java.base/sun.security.ssl.SSLSocketImpl$AppInputStream.read(SSLSocketImpl.java:839)
	at java.base/sun.nio.cs.StreamDecoder.readBytes(StreamDecoder.java:284)
	at java.base/sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:326)
	at java.base/sun.nio.cs.StreamDecoder.read(StreamDecoder.java:178)
	at java.base/java.io.InputStreamReader.read(InputStreamReader.java:185)
	at java.base/java.io.BufferedReader.fill(BufferedReader.java:161)
	at java.base/java.io.BufferedReader.read1(BufferedReader.java:212)
	at java.base/java.io.BufferedReader.read(BufferedReader.java:287)
	at org.xmlpull.mxp1.MXParser.fillBuf(MXParser.java:2992)
	at org.xmlpull.mxp1.MXParser.more(MXParser.java:3046)
	at org.xmlpull.mxp1.MXParser.nextImpl(MXParser.java:1144)
	at org.xmlpull.mxp1.MXParser.next(MXParser.java:1093)
	at org.jivesoftware.smack.PacketReader.parsePackets(PacketReader.java:312)
	at org.jivesoftware.smack.PacketReader.access$000(PacketReader.java:48)
	at org.jivesoftware.smack.PacketReader$1.run(PacketReader.java:76)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.base/java.net.SocketInputStream.socketRead0(Native Method)
	at java.base/java.net.SocketInputStream.socketRead(SocketInputStream.java:115)
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:168)
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:140)
	at java.base/sun.security.ssl.SSLSocketInputRecord.read(SSLSocketInputRecord.java:448)
	at java.base/sun.security.ssl.SSLSocketInputRecord.bytesInCompletePacket(SSLSocketInputRecord.java:68)
	at java.base/sun.security.ssl.SSLSocketImpl.readApplicationRecord(SSLSocketImpl.java:1104)
	at java.base/sun.security.ssl.SSLSocketImpl$AppInputStream.read(SSLSocketImpl.java:823)
	... 14 more

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-26 22:14:56.344
!MESSAGE  new connection state == ERROR (XMPPConnectionService.java:400)
!STACK 0
javax.net.ssl.SSLProtocolException: Software caused connection abort: recv failed
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:126)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:321)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:264)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:259)
	at java.base/sun.security.ssl.SSLSocketImpl.handleException(SSLSocketImpl.java:1314)
	at java.base/sun.security.ssl.SSLSocketImpl$AppInputStream.read(SSLSocketImpl.java:839)
	at java.base/sun.nio.cs.StreamDecoder.readBytes(StreamDecoder.java:284)
	at java.base/sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:326)
	at java.base/sun.nio.cs.StreamDecoder.read(StreamDecoder.java:178)
	at java.base/java.io.InputStreamReader.read(InputStreamReader.java:185)
	at java.base/java.io.BufferedReader.fill(BufferedReader.java:161)
	at java.base/java.io.BufferedReader.read1(BufferedReader.java:212)
	at java.base/java.io.BufferedReader.read(BufferedReader.java:287)
	at org.xmlpull.mxp1.MXParser.fillBuf(MXParser.java:2992)
	at org.xmlpull.mxp1.MXParser.more(MXParser.java:3046)
	at org.xmlpull.mxp1.MXParser.nextImpl(MXParser.java:1144)
	at org.xmlpull.mxp1.MXParser.next(MXParser.java:1093)
	at org.jivesoftware.smack.PacketReader.parsePackets(PacketReader.java:312)
	at org.jivesoftware.smack.PacketReader.access$000(PacketReader.java:48)
	at org.jivesoftware.smack.PacketReader$1.run(PacketReader.java:76)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.base/java.net.SocketInputStream.socketRead0(Native Method)
	at java.base/java.net.SocketInputStream.socketRead(SocketInputStream.java:115)
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:168)
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:140)
	at java.base/sun.security.ssl.SSLSocketInputRecord.read(SSLSocketInputRecord.java:448)
	at java.base/sun.security.ssl.SSLSocketInputRecord.bytesInCompletePacket(SSLSocketInputRecord.java:68)
	at java.base/sun.security.ssl.SSLSocketImpl.readApplicationRecord(SSLSocketImpl.java:1104)
	at java.base/sun.security.ssl.SSLSocketImpl$AppInputStream.read(SSLSocketImpl.java:823)
	... 14 more
!SESSION 2019-05-27 15:38:17.068 -----------------------------------------------
eclipse.buildId=4.11.0.I20190307-0500
java.version=11.0.2
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=de_DE
Framework arguments:  -product org.eclipse.epp.package.java.product
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.java.product

!ENTRY org.eclipse.egit.core 1 0 2019-05-27 15:38:27.847
!MESSAGE Using Apache MINA sshd as ssh client.

!ENTRY org.eclipse.egit.ui 2 0 2019-05-27 15:39:04.717
!MESSAGE Warning: The environment variable HOME is not set. The following directory will be used to store the Git
user global configuration and to define the default location to store repositories: 'C:\Users\Jannik'. If this is
not correct please set the HOME environment variable and restart Eclipse. Otherwise Git for Windows and
EGit might behave differently since they see different configuration options.
This warning can be switched off on the Team > Git > Confirmations and Warnings preference page.

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-27 15:39:18.489
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-27 15:39:18.501
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-27 15:39:18.507
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-27 15:39:32.340
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.178.20:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-27 15:39:32.880
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /169.254.181.9:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-27 15:39:32.883
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.38.1:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-27 15:39:32.890
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.77.1:0 (StunServiceImpl.java:420)
!SESSION 2019-05-30 16:47:22.178 -----------------------------------------------
eclipse.buildId=4.11.0.I20190307-0500
java.version=11.0.2
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=de_DE
Framework arguments:  -product org.eclipse.epp.package.java.product
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.java.product

!ENTRY org.eclipse.core.resources 2 10035 2019-05-30 16:47:29.648
!MESSAGE The workspace exited with unsaved changes in the previous session; refreshing workspace to recover changes.

!ENTRY org.eclipse.egit.core 1 0 2019-05-30 16:47:47.305
!MESSAGE Using Apache MINA sshd as ssh client.

!ENTRY org.eclipse.egit.ui 2 0 2019-05-30 16:48:23.058
!MESSAGE Warning: The environment variable HOME is not set. The following directory will be used to store the Git
user global configuration and to define the default location to store repositories: 'C:\Users\Jannik'. If this is
not correct please set the HOME environment variable and restart Eclipse. Otherwise Git for Windows and
EGit might behave differently since they see different configuration options.
This warning can be switched off on the Team > Git > Confirmations and Warnings preference page.

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-30 16:48:27.312
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-30 16:48:27.320
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-30 16:48:27.348
!MESSAGE error sending STUN request (StunServiceImpl.java:263)
!STACK 0
java.net.SocketException: Network is unreachable: Datagram send failed
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketSend(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.send(DualStackPlainDatagramSocketImpl.java:136)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:695)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery$1.run(StunServiceImpl.java:255)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 4 0 2019-05-30 16:48:27.943
!MESSAGE an error occurred while performing a STUN discovery: ICMP Port Unreachable (StunServiceImpl.java:155)
!STACK 0
java.net.PortUnreachableException: ICMP Port Unreachable
	at java.base/java.net.DualStackPlainDatagramSocketImpl.socketReceiveOrPeekData(Native Method)
	at java.base/java.net.DualStackPlainDatagramSocketImpl.receive0(DualStackPlainDatagramSocketImpl.java:124)
	at java.base/java.net.AbstractPlainDatagramSocketImpl.receive(AbstractPlainDatagramSocketImpl.java:181)
	at java.base/java.net.DatagramSocket.receive(DatagramSocket.java:814)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery.performStunDiscovery(StunServiceImpl.java:284)
	at de.fu_berlin.inf.dpp.net.stun.internal.StunServiceImpl$StunDiscovery.run(StunServiceImpl.java:151)
	at java.base/java.lang.Thread.run(Thread.java:834)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-30 16:48:37.309
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /169.254.181.9:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-30 16:48:37.319
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.77.1:0 (StunServiceImpl.java:420)

!ENTRY de.fu_berlin.inf.dpp 2 0 2019-05-30 16:48:37.322
!MESSAGE received no response from STUN server stunserver.org/67.227.226.240:3478 at local address /192.168.38.1:0 (StunServiceImpl.java:420)
